draft-js/draft-js-tests-0.2.2.tsx

```diff
@@ -0,0 +1,183 @@
+/// <reference path="draft-js-0.2.2.d.ts" />
+/// <reference path="../react/react-global.d.ts" />
+/// <reference path="../react/react-dom.d.ts"/>
+
+import * as React from "react";
+
+import {
+Editor,
```

Comment:
@zhengbli, I'm new to definitelyTyped and might miss something obvious. Did you mean the `Editor` in the updated definition file (i.e. draft-js.d.ts)?

The definition file for new version (i.e. draft-js.d.ts) contains the definition for `Editor`.
For instance, test file (draft-js.tests.tsx) imports the definition for `Editor` and uses it.